sdk/js/src/service/join-server.js

```diff
@@ -0,0 +1,30 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Marshaler from '../util/marshaler'
+
+class Js {
+  constructor (service) {
+    this._api = service
+  }
+
+  async listJoinEUIPrefixes () {
+    const result = await this._api.GetJoinEUIPrefixes()
+    const { prefixes } = Marshaler.payloadListResponse('prefixes', result)
```

Comment:
Any reason to no simply return the marshaled value, like we do in other services?
```suggestion
    return Marshaler.payloadListResponse('prefixes', result)
```

Reply:
we are not paginating prefixes, there is no reason to return the `totalCount` entry. even we decide to implement pagination for this, it will be an api change

Reply:
True. We should refactor the configuration service then as well.

Reply:
Actually, it makes sense to include the `totalCount` entry if we do this for the configuration service